ansible/roles/stack-sunbird/templates/inbound.env

```diff
@@ -51,3 +51,8 @@ REDIS_HOST={{sunbird_redis_host}}
 REDIS_PASS={{sunbird_redis_pass | default('')}}
 REDIS_PORT={{sunbird_redis_port | default(6379)}}
 REDIS_DB_INDEX={{redis_db_index_uci | default('7')}}
+
+#Azure Config
+AZURE_BLOB_STORE_CONTAINER={{sunbird_azure_uci_container_name}}
```

Comment:
Please add a default value like below so that we can avoid adding additonal variables
`{{sunbird_azure_uci_container_name | default('uci-' + env )}}`